ci/checksizes.sh

```diff
@@ -34,20 +34,20 @@ REVERSI_OPT_GZ_SIZE=$(stat '-c%s' "$REVERSI_OPT.gz")
 
 case $FULLVER in
   2.11.12)
-    REVERSI_PREOPT_EXPECTEDSIZE=439000
-    REVERSI_OPT_EXPECTEDSIZE=95000
+    REVERSI_PREOPT_EXPECTEDSIZE=436000
+    REVERSI_OPT_EXPECTEDSIZE=94000
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=59000
     REVERSI_OPT_GZ_EXPECTEDSIZE=27000
     ;;
   2.12.10)
-    REVERSI_PREOPT_EXPECTEDSIZE=412000
+    REVERSI_PREOPT_EXPECTEDSIZE=409000
     REVERSI_OPT_EXPECTEDSIZE=90000
     REVERSI_PREOPT_GZ_EXPECTEDSIZE=54000
     REVERSI_OPT_GZ_EXPECTEDSIZE=25000
     ;;
   2.13.1)
-    REVERSI_PREOPT_EXPECTEDSIZE=557000
-    REVERSI_OPT_EXPECTEDSIZE=123000
+    REVERSI_PREOPT_EXPECTEDSIZE=554000
+    REVERSI_OPT_EXPECTEDSIZE=122000
```

Comment:
The changes in checksizes are new, with improved sizes (whereas before fastOpt was increasing by about 5KB).